spec/services/voucher_adjustments_service_spec.rb

```diff
@@ -53,18 +51,14 @@
 
       it 'updates the adjustment included_tax' do
         # voucher_rate = amount / order.total
-        # -10 / 150 = -0.066666667
+        # -10 / 160 = -0.0625
         # included_tax = voucher_rate * order.included_tax_total
-        # -0.66666666 * 10 = -0.67
-        expect(subject.included_tax.to_f).to eq(-0.67)
-      end
-
-      it 'moves the adjustment state to closed' do
-        expect(subject.state).to eq('closed')
+        # -0.625 * 10 = -0.63
+        expect(subject.included_tax.to_f).to eq(-0.63)
```

Comment:
:question: 
May brain computes this:
```
-0.625 * 10 = -6.25
```

The order total is 110, the included tax is 10. The voucher rate is 0.0625, ah, so it should be:
```suggestion
        # -0.0625 * 10 = -0.625
        expect(subject.included_tax.to_f).to eq(-0.63)
```

The rounding can then be observed.